config/initializers/sentry.rb

```diff
@@ -0,0 +1,12 @@
+require 'active_support/parameter_filter'
+
+Sentry.init do |config|
+  config.breadcrumbs_logger = [:active_support_logger, :http_logger]
+
+  config.traces_sample_rate = 1.0
+
+  filter = ActiveSupport::ParameterFilter.new(Rails.application.config.filter_parameters)
+  config.before_send = lambda do |event, _hint|
+    filter.filter(event.to_hash)
+  end
```

Comment:
I recommend that we prevent Sentry from logging for 422 and "user errors," which can be done with an excluded exceptions array like this:

https://github.com/DFE-Digital/find-a-lost-trn/blob/main/config/initializers/sentry.rb#L26-L34

Reply:
Would it maybe be worth keeping these while we're in the model office & pilot phase? These could be interesting, and it's hidden behind auth so we shouldn't get spurious script kiddies and vulnerability scanning.

Reply:
Hmm, yeah, we can revisit this later :+1: